packages/release/release-fips-tmpfiles.conf

```diff
@@ -0,0 +1,2 @@
+f+ /etc/system-fips 0644 root root -
+r /etc/.fips-modprobe.*
```

Comment:
We end up writing at least one error to a file like this for every module. We only care about fatal errors from modprobe, which will halt the system before we get to the point of running `systemd-tmpfiles`. So this just cleans up the garbage with no diagnostic value.